spec/features/request_to_book_spec.rb

```diff
@@ -1,26 +1,31 @@
 feature 'Request space' do
 
+  before :each do
+    host = db_create_user("dude","d@m.com","dude","dude_pass")
```

Comment:
Serious people read these tests, they are part of our CV, I think there are better words than 'dude'. Lets maybe use Test_user or something in that direction